htdocs/js/Problem/problem.scss

```diff
@@ -123,6 +123,12 @@
 			border: 1px solid black;
 		}
 	}
+
+	/* Hints and Solutions */
+	.hint summary, .solution summary {
+		font-weight: bold;
+		color: var(--bs-primary);
```

Comment:
Using the `--bs-primary` color has contrast issues for the `math4-yellow` theme.  Instead you should use the primary text color.  That will be the same for all of the other themes, but for `math4-yellow` is a darker color.  Unfortunately, there isn't a css variable that contains that.  However, this can still be attained by deleting the `color` setting here, and instead adding the class `text-primary` to either the `summary.accordion-button` element or the `div.accordion-header` element inside that.

You could also add the `fw-bold` class to the same element and then you don't need the addition here, but either way is the same for that.